docs/reference/display.md

```diff
@@ -733,4 +733,148 @@ The `wb_display_image_delete` function releases the memory used by a clipboard i
 After this call the value of `ir` becomes invalid and should not be used any more.
 Using this function is recommended after a clipboard image is not needed any more.
 
-> **Note** [Java]: The `Display.imageNew` function can display the image returned by the `Camera.getImage` function directly if the pixel format argument is set to ARGB.
+The following controller snippet shows how to copy a [Camera](camera.md) image to the main display image.
+This is particularly useful if you need to process the [Camera](camera.md) image before copying it to the main display image, otherwise please use the [`wb_display_attach_camera`](#wb_display_attach_camera) function that provides better performance.
+
+%tab-component "language"
+%tab "C"
+```c
+#include <webots/robot.h>
+#include <webots/camera.h>
+#include <webots/display.h>
+
+int main() {
+  wb_robot_init();
+
+  int time_step = wb_robot_get_basic_time_step();
+  WbDeviceTag camera = wb_robot_get_device("camera");
+  wb_camera_enable(camera, time_step);
+  const int width = wb_camera_get_width(camera);
+  const int height = wb_camera_get_height(camera);
+  WbDeviceTag display = wb_robot_get_device("display");
+
+  while(wb_robot_step(time_step) != -1) {
+    const unsigned char *data = wb_camera_get_image(camera);
+    if (data) {
+      WbImageRef ir = wb_display_image_new(display, width, height, data, WB_IMAGE_ARGB);
```

Comment:
The `wb_camera_get_image` returns data in BGRA format.
But to work correctly I have to set the `wb_display_image_new` format to ARGB.
This doesn't seem very intuitive for users.

Reply:
The inversion comes from this line:
https://github.com/cyberbotics/webots/blob/fb5cd3df739e4773aa677d9445eebd2ab849493f/src/Controller/api/display.c#L751

For `wb_camera_get_image` input data in BGRA format the image pixel uint32_t is `0xbbggrraa`.
But it seems that internally it is stored in little-endian because accessing the image bytes you get inversed values and inversed bytes are sent to Webots ([here](https://github.com/cyberbotics/webots/blob/master/src/Controller/api/request.c#L113-L118)).
```
printf("pixel %02x %02x %02x %02x\n", ((unsigned char *)(i->image))[0], ((unsigned char *)(i->image))[1], ((unsigned char *)(i->image))[2], ((unsigned char *)(i->image))[3]);
# result for pixel data B=152(0x98) G=109(0x6d) R=90(0x5a) A=255(0xff)
# "pixel ff 5a 6d 98"
```

I fixed it locally and run the test suite and everything was working correctly.
But I would fix this on the `develop` branch to prevent issues with user controllers.